webpack/scenes/AlternateContentSources/Create/Steps/ACSReview.js

```diff
@@ -76,7 +76,7 @@ const ACSReview = () => {
                   </TextListItem>
                   <TextListItem component={TextListItemVariants.dt}>{__('Password')}</TextListItem>
                   <TextListItem component={TextListItemVariants.dd}>
-                    {password}
+                    {password.length > 0 ? '••••••••' : ''}
```

Comment:
This looks better IMO. Let me know what you think.
```suggestion
                    {password.length > 0 ? '••••••••' : <InactiveText text={__('N/A')}/>}
```
Also add `import InactiveText from "../../../ContentViews/components/InactiveText";` to the end of the import section of this file if you make this change.